cmd/build/helmify/static/README.md

```diff
@@ -7,6 +7,7 @@
 | auditInterval             | The frequency with which audit is run                       | `60`                                                                      |
 | constraintViolationsLimit | The maximum # of audit violations reported on a constraint  | `20`                                                                      |
 | auditFromCache            | Take the roster of resources to audit from the OPA cache    | `false`                                                                   |
+| auditChunkSize            | (alpha) Chunk size for listing cluster resources for audit  | `0`                                                                       |
```

Comment:
can we make sure the alpha notation is consistent here. can you update the other ones to prefix with `(alpha)` as well?